### ISSUE TITLE
[🐛 Bug]: isStable method fails if test executes in the browser which is in the background or in the tray

### ISSUE THREAD

**Issue by HannaTarasevich** [webdriverio/webdriverio#11995]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

latest

### Node.js Version

latest

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
{
  "name": "my-new-project",
  "type": "module",
  "devDependencies": {
    "@wdio/cli": "^8.27.2",
    "@wdio/cucumber-framework": "^8.27.2",
    "@wdio/dot-reporter": "^8.23.0",
    "@wdio/local-runner": "^8.27.2",
    "@wdio/spec-reporter": "^8.27.2",
    "chai": "^4.3.10",
    "wdio-wait-for": "^3.0.9"
  },
  "scripts": {
    "wdio": "wdio run ./wdio.conf.js"
  }
}
```

### What happened?

Suppose the tests are running in the browser in the background (fully covered by any other window), or the browser with running tests is minimized to tray. In that case, isStable method fails with an error `AssertionError: javascript error: You are are checking for animations on an inactive tab, animations do not run for inactive tabs`.

It would be pretty much useful not to fail the method, since actually animations are working in the application even in such cases (e.g., some animation after clicking on other elements).

```
    if (document.visibilityState === 'hidden') {
        throw Error('You are are checking for animations on an inactive tab, animations do not run for inactive tabs')
    }

```

### What is your expected behavior?

isStable is working even if browser tests execution is in the background or in the tray.

### How to reproduce the bug.

https://github.com/HannaTarasevich/wdio-timeout-issue

Steps:
1) npm run wdio
2) minimize appeared Chrome for testing browser to the tray right after it appears

### Relevant log output

```typescript
[chrome 120.0.6099.217 windows #0-0] 1) As a user, I can log into the secure area When I login with tomsmith and SuperSecretPassword!
[chrome 120.0.6099.217 windows #0-0] Error: javascript error: You are are checking for animations on an inactive tab, animations do not run for inactive tabs
[chrome 120.0.6099.217 windows #0-0] Error: javascript error: You are are checking for animations on an inactive tab, animations do not run for inactive tabs
[chrome 120.0.6099.217 windows #0-0] JavaScript stack:
[chrome 120.0.6099.217 windows #0-0] Error: You are are checking for animations on an inactive tab, animations do not run for inactive tabs
[chrome 120.0.6099.217 windows #0-0]     at isElementStable (eval at executeAsyncScript (:447:26), <anonymous>:5:15)
[chrome 120.0.6099.217 windows #0-0]     at eval (eval at executeAsyncScript (:447:26), <anonymous>:25:4)
[chrome 120.0.6099.217 windows #0-0]     at eval (eval at executeAsyncScript (:447:26), <anonymous>:26:4)
[chrome 120.0.6099.217 windows #0-0]     at executeAsyncScript (<anonymous>:447:47)
[chrome 120.0.6099.217 windows #0-0]     at apply.ELEMENT (<anonymous>:462:29)
[chrome 120.0.6099.217 windows #0-0]     at callFunction (<anonymous>:341:22)
[chrome 120.0.6099.217 windows #0-0]     at <anonymous>:355:23
[chrome 120.0.6099.217 windows #0-0]     at <anonymous>:356:3
[chrome 120.0.6099.217 windows #0-0]   (Session info: chrome=120.0.6099.217)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
@erwinheitzman what do you think?

**Comment by BorisOsipov** [comment]
>It would be pretty much useful not to fail the method, since actually animations are working in the application even in such cases (e.g., some animation after clicking on other elements like animated dropdown menu appearing).

No. If the browser tab is inactive, the real browser may slow down or even stop the animation.

We cannot detect if there is an active animation due to [Policies in place to aid background page performance](https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API#policies_in_place_to_aid_background_page_performance)
```
Most browsers stop sending `requestAnimationFrame()` callbacks to background tabs or hidden in order to improve performance and battery life.
```

So if we remove the check

```
    if (document.visibilityState === 'hidden') {
        throw Error('You are are checking for animations on an inactive tab, animations do not run for inactive tabs')
    }
```

You just get weird javacript timeout error instead of actual meaningful error.

**Comment by HannaTarasevich** [comment]
@BorisOsipov any ideas how the script can be improved?

**Comment by BorisOsipov** [comment]
@HannaTarasevich 

Improved in what direction? We can detect animation only via `requestAnimationFrame()` callback. Such callback never calls in hidden browser.

If we ignore that the browser is hidden and return true in such a case - we lie to people because actually animation may still be in progress and vice-versa.

I think there may some chromium\ff browser startup args to prevent background trolling, but that is outside the scope of wdio and this script.

**Comment by BorisOsipov** [comment]
E.g. for chromium it seems `--disable-renderer-backgrounding` arg.

**Comment by HannaTarasevich** [comment]
Got it, thanks. Looks like there is no way to use the method for such cases
Animations are working with visibilityState = hidden for my case, so the flag won't help

**Comment by erwinheitzman** [comment]
@HannaTarasevich do you feel like we still need changes here? I don't think it makes sense to check state in an inactive tab but I might be overlooking use cases

**Comment by jendaprokes-enverus** [comment]
adding chrome option `--disable-backgrounding-occluded-windows` has fixed it in my case when the e2e tests were failing locally in headless mode: https://github.com/GoogleChrome/chrome-launcher/blob/main/docs/chrome-flags-for-tools.md?plain=1#L29

**Comment by christian-bromann** [comment]
Should we add this to the `isStabel` command docs?

**Comment by alek75** [comment]
Just FYI: The error also occurs when run the tests is headless mode.

The proposed fix is working.
Cheers

**Comment by christian-bromann** [comment]
Added a commit that enhances the docs of the `isStable` command for this problem.
